server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -619,6 +624,42 @@ export function configureWebSocketServices(
 				);
 			}
 
+			// Set up listener to forward signal to clients in the collaboration session when the broadcast-signal endpoint is called
+			collborationSessionEventEmitter?.on(
+				"broadcastSignal",
+				(broadcastSignal: IBroadcastSignalEventPayload) => {
```

Comment:
We already define other interfaces to represent a tenantId/documentId combination. For example, in line 384:
```ts
const room: IRoom = {
				tenantId: claims.tenantId,
				documentId: claims.documentId,
			};
```
Would it make sense to have `IBroadcastSignalEventPayload` include an `IRoom` property instead of having separate `documentId` and `tenantId` properties?

Reply:
That could work!